contracts/system_addresses.go

```diff
@@ -39,6 +39,12 @@ var (
 	// RootMintableERC1155PredicateContract is an address of mintable ERC1155 predicate on the child chain
 	RootMintableERC1155PredicateContract = types.StringToAddress("0x100b")
 
+	// Governance contracts
+	ChildGovernorContract = types.StringToAddress("0x100c")
```

Comment:
Missing more consistent comments here but it can be done in upcoming PRs, just note it